lib/pbench/server/api/resources/__init__.py

```diff
@@ -1770,6 +1770,7 @@ def _dispatch(
             current_app.logger.exception("{} {}", api_name, e.details)
             abort(e.http_status, message=str(e))
         except APIAbort as e:
+            current_app.logger.warning("{} {}", api_name, e)
```

Comment:
Before we eradicated all "user error" logs, we had established a convention of reserving "WARNING" for those, which separates them from innocuous internal "INFO" progress messages. I wish we had a separate label, and in theory `logging` provides the capability to define them; but I didn't want to get into that before, and I don't particularly now.

Reply:
I'm certainly sympathetic to the resistance to creating a new label at this point.  And, I do have some sort of vague recollection of reserving `WARNING` for user problems (and, I can see how you would want to separate user problems from Server "informational" messages).  But, it still feels wrong to have these messages at the `WARNING` level...aren't they intended for debugging?...why not make them `DEBUG` level?  (Hopefully, they won't be needlessly distracting.)

Reply:
They're not for code debugging: they're for ops review debugging, as in, all the times we found ourselves saying "oh, nginx says we returned a 400: why?" Having the UI "user error" logs clearly distinguished from normal internal `info` messages is extremely valuable for filtering. I suppose that since I consolidated the generation in `_dispatch` I could have invented a clumsy text marker approach; and I might have done that aside from the fact that we *had* all agreed long ago on `warning` for this purpose and that's easier to filter.